lib/WebRtcPeer.js

```diff
@@ -917,4 +917,4 @@ exports.bufferizeCandidates = bufferizeCandidates
 exports.WebRtcPeerRecvonly = WebRtcPeerRecvonly
 exports.WebRtcPeerSendonly = WebRtcPeerSendonly
 exports.WebRtcPeerSendrecv = WebRtcPeerSendrecv
-exports.hark = harkUtils
+exports.hark = harkUtils
```

Comment:
Why is this change sneaking here? What is the purpose of exporting hark from this file?